### PR TITLE
WT-10647 RTS perf characterisation

### PR DESCRIPTION
**Pull request by korteland**
This tweaks workgen to add RTS as a timed operation. There are also some microbenchmarks that were used to generate some performance numbers (see the wiki page linked on the ticket). These tests aren't appropriate to run in Evergreen so we haven't added them.

### CODE REVIEW

**Comment by ddanderson** [comment]
@korteland, the description of this PR says to look at the ticket to see a wiki page linked.  I don't see it.

File: bench/workgen/runner/microbenchmark_rts_unstable_content.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    context = Context()
+
+    conn = context.wiredtiger_open("create")
+    conn.set_timestamp('stable_timestamp=' + timestamp_str(5))
+    session = conn.open_session()
+    uri = 'table:rts_unstable_content'
+    session.create(uri, 'key_format=S, value_format=S')
+
+    inserts = 10
+    ops = Operation(Operation.OP_CHECKPOINT, "")
```
Comment (ddanderson) [inline bench/workgen/runner/microbenchmark_rts_unstable_content.py:62]: This looks like a stray line?

**Review by ddanderson — APPROVED** [review]
This looks great, and a nice use case for workgen.

**Comment by korteland** [comment]
Oops, sorry about the wiki page link - I've added that now.

File: bench/workgen/runner/microbenchmark_rts_unstable_content.py (hunk 1 above)
Comment (korteland) [reply bench/workgen/runner/microbenchmark_rts_unstable_content.py]: It is - thanks, fixed in df19f8e.

**Review by clarissecheah — APPROVED** [review]
LGTM! :)

**Pull request merged**